corehq/apps/reports/tests/sql_fixture.py

```diff
@@ -35,25 +33,37 @@ def load_data():
     metadata.create_all()
 
     user_data = [
-        {"user": "user1", "date": date(2013, 1, 1), "indicator_a": 1, "indicator_b": 0, "indicator_c": 1, "indicator_d": 1},
-        {"user": "user1", "date": date(2013, 2, 1), "indicator_a": 0, "indicator_b": 1, "indicator_c": 1, "indicator_d": 1},
-        {"user": "user2", "date": date(2013, 1, 1), "indicator_a": 0, "indicator_b": 1, "indicator_c": 1, "indicator_d": 2},
-        {"user": "user2", "date": date(2013, 2, 1), "indicator_a": 1, "indicator_b": 0, "indicator_c": 1, "indicator_d": 2},
+        {"user": "user1", "date": date(2013, 1, 1),
```

Comment:
The new code is harder to read.

Reply:
Agreed, but I don't think there's a great way around this -- this warning was based on line length. Do you think this is worth an explicit exception?

Reply:
Yes. A `# noqa` at the end of each line would do it, right? I think that would be pretty unobtrusive.